### DISCUSSION TITLE
How can I translate the document into Japanese?

### DISCUSSION THREAD

**Discussion by tamagoez** [saadeghi/daisyui#595] [Q&A] [answered]
I am Japanese.
I ❤️ this component, so I want to contribute to this repositories doccument.
How can I translate it into Japanese?

**Comment by wuarmin** [comment, marked as answer]
Hey,
checkout https://github.com/saadeghi/daisyui/blob/master/src/docs/src/translation/jp.json

**Reply by tamagoez** [reply]
Thank you.

**Comment by saadeghi** [comment]
Currently there are 2 main jobs to do:  
1. Fixing currently translated strings. This is easier  
Edit the [translation file](https://github.com/saadeghi/daisyui/blob/master/src/docs/src/translation/jp.json) that @wuarmin mentioned which was translated using Google Translate API initially.  

2. Translate new strings. This takes more time  
    1. Add the translate function to the pages  
You can find `*.svelte.md` files here on [`/docs`](https://github.com/saadeghi/daisyui/tree/master/src/docs/src/routes/docs) and [`/components`](https://github.com/saadeghi/daisyui/tree/master/src/docs/src/routes/components)  
          There's a `translate` Svelte component that you can import (like [this](https://github.com/saadeghi/daisyui/blob/25dc1bcd61ca1084eb5d55a37111b55ecd4b97ac/src/docs/src/routes/docs/install.svelte.md?plain=1#L9)):
          ```svelte
          <script>
            import Translate from "@components/Translate.svelte"
          </script>
          ```
      2. replace raw strings inside `*.svelte.md` files.
          from:
          ```svelte
          hello world
          ```
          to:
          ```svelte
          <Translate text="hello world" />
          ```
      3. Add the new `hello world` string to [translation file](https://github.com/saadeghi/daisyui/blob/master/src/docs/src/translation/jp.json):  
          ```json
          "hello world": "こんにちは世界"
          ```

Please avoid large PRs with multiple files edits to avoid conflicts.

ありがとうございます 🙌
